src/names.c

```diff
@@ -708,17 +708,32 @@ SEXP vec_set_rownames(SEXP x, SEXP names) {
   return x;
 }
 
+SEXP vec_set_df_rownames(SEXP x, SEXP names) {
+  x = PROTECT(r_maybe_duplicate(x));
+
+  if (names == R_NilValue) {
+    init_compact_rownames(x, vec_size(x));
+  } else {
+    // Repair row names verbosely
```

Comment:
You say verbose here, but silent in the news bullet? Did I read something wrong? The default unique repair opts seem to have `quiet = false`

Reply:
It should say "verbosely" in NEWS (but without error).